lib/bot.js

```diff
@@ -144,9 +144,18 @@ class Bot {
         this.ircClient.say(ircChannel, prelude);
         this.ircClient.say(ircChannel, text);
       } else {
-        text = `<${coloredUsername}> ${text}`;
-        logger.debug('Sending message to IRC', ircChannel, text);
-        this.ircClient.say(ircChannel, text);
+        if (text !== '') {
+          text = `<${coloredUsername}> ${text}`;
+          logger.debug('Sending message to IRC', ircChannel, text);
+          this.ircClient.say(ircChannel, text);
+        }
+        if (message.attachments) {
+          message.attachments.forEach(a => {
+            const urlMessage = `<${coloredUsername}> ${a.url}`;
```

Comment:
Maybe it would be better to post a more human friendly description here?
Something like: `Bob has posted an attachment to #channel on Discord: <link>` or similar.